TypeCobol/Compiler/CupCommon/CobolWordsTokenizer.cs

```diff
@@ -271,7 +271,7 @@ public void EnterStopScanningModeIfNextNotToken(TokenType nextTokenType)
                 return;//Ignore if end of file
             Token nextToken = base.NextToken();
             base.PreviousToken();
-            if (nextToken != null && nextToken.TokenType != nextTokenType)
+            if (nextToken != null && nextTokenType.All(t=>t!=nextToken.TokenType))
```

Comment:
Could you add spaces around '=>' ?

Reply:
Done in e705b29f87cea7244fae092d8bc7488e8de90876